src/serialization/pkl2proto.py

```diff
@@ -0,0 +1,155 @@
+import pickle
+import sys
+import numpy as np
+import font_pb2
+import serialization 
+
+def parseDict(d, filename= "No filename provided"):
```

Comment:
It's more Pythonic to let default arguments be `None`, and let them be caught and assigned later. Setting `filename="No filename provided."` can lead to some unexpected behavior. But in the interest of time I won't insist on it.

Reply:
cc @ostapstephan